### DISCUSSION TITLE
Access token regneration

### DISCUSSION THREAD

**Discussion by RedKnight21** [community/community#30862] [Code Security]
I received this e-mail a few days ago:

Hi @RedKnight21,

We noticed your personal access token "iMac 29 August 2021" with repo scope will expire in 6 days.

If this token is still needed, visit https://github.com/settings/tokens/683667276/regenerate to generate an equivalent.

If you run into problems, please contact support by visiting https://github.com/contact?tags=dotcom-accounts

The link to regenerate does not load the web page.  Don't know what to do from here.

RedKnight21

**Comment by airtower-luna** [comment]
Log in manually, then got to your user settings. Choose "Developer Settings" and then "Personal access tokens". You should see a list of your tokens. Click the one you want, and you should have the option to regenerate it. :slightly_smiling_face: Mind that "regenerate" means "create a new token with the same permissions", so you'll then need to update the token wherever you use it.

And general advice: Don't log in via links in an email. Sometimes recognizing phishing mails is hard, logging in manually via a known URL is much safer. :warning:

**Comment by tenmark86** [comment]
@RedKnight21 just go to your settings then scroll to tokens section you'll see the expired token click on it then click on regenerate tab.
